ocf_datapipes/load/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .gsp import OpenGSPIterDataPipe as OpenGSP
```

Comment:
Are we sure we want these in the init file? I think that means it automatically loads all of them, even thought we might not be using them?